### PR TITLE
Removed step function report file

### PR DESCRIPTION
**Pull request by hhm970**
Dan told us we don't need a step function for the weekly report, so I deleted the step function. Updated `report_eventbridge.tf` to account for this.

Let me know if there's anything I missed with my approach!

### CODE REVIEW

**Review by annalisev — COMMENTED** [review]
Yes this makes sense, but have you updated the terraformed eventbridge to now run a lambda now?

File: database/schema.sql [hunk 1]
```diff
@@ -9,7 +9,7 @@ DROP TABLE tag CASCADE;
 DROP TABLE game CASCADE;
 DROP TABLE platform_assignment CASCADE;
 DROP TABLE game_tag_matching CASCADE;
-DROP TABLE subscriber
+DROP TABLE subscriber;
```
Comment (annalisev) [inline database/schema.sql:12]: good spot!

File: terraform/report_eventbridge.tf [hunk 2]
```diff
@@ -9,7 +9,7 @@ resource "aws_scheduler_schedule" "c10-games-terraform-report" {
   schedule_expression = "cron(0 9 ? * 2 *)"
 
   target {
-    arn      = "arn:aws:states:eu-west-2:129033205317:stateMachine:c10-games-statemachine-terraform-report"
-    role_arn = aws_iam_role.eventbridge-role.arn
+    arn      = "arn:aws:lambda:eu-west-2:129033205317:function:c10-games-terraform-weekly-report"
+    role_arn = aws_iam_role.lambda-role.arn
```
Comment (annalisev) [inline terraform/report_eventbridge.tf:13]: cool looks good!

**Review by annalisev — APPROVED** [review]
Looks good Howaard!

**Comment by shindym** [comment]
Looks good!

**Review by shindym — APPROVED** [review]

File: database/schema.sql (hunk 1 above)
Comment (hhm970) [reply database/schema.sql:12]: Credit to @Eibironke for this

**Pull request merged**